src/Types.jl

```diff
@@ -0,0 +1,258 @@
+export TheoryPetriNet, PetriNet, AbstractPetriNet, ns, nt, ni, no,
+  add_species!, add_transition!, add_transitions!,
+  add_input!, add_inputs!, add_output!, add_outputs!, inputs, outputs,
+  TransitionMatrices, vectorfield,
+  TheoryLabelledPetriNet, LabelledPetriNet, AbstractLabelledPetriNet, sname, tname,
+  TheoryReactionNet, ReactionNet, AbstractReactionNet, concentration, concentrations, rate, rates,
+  TheoryLabelledReactionNet, LabelledReactionNet, AbstractLabelledReactionNet
+
+using Catlab
+using Catlab.CategoricalAlgebra.CSets
+using Catlab.Present
+using Catlab.Theories
+using Petri
+using LinearAlgebra: mul!
+import Petri: Model
+
+vectorify(n) = begin
```

Comment:
Is this function needed? The C-sets API is vectorized, so you should be able to mix scalars and vectors without worrying about it.

Reply:
Ah this helper function is for the constructors to help me iterate over both arrays and scalars. I was running into some weird behavior where I was getting a 0-dimensional array in one of the constructors. It's not to help with the C-Set API